egs/vcc2020/README.md

```diff
@@ -0,0 +1,63 @@
+
+The training flow is as follows:
+
+1. ASR training. A [Transformer-based ASR model](https://github.com/espnet/espnet/tree/master/egs/librispeech/asr1) is used. ESPnet provides a [LibriSpeech pretrained model](https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md#pytorch-large-transformer-with-specaug-4-gpus--large-lstm-lm).
+2. TTS pretraining. We use a [multi-speaker, x-vector Transformer-TTS model](https://github.com/espnet/espnet/tree/master/egs/libritts/tts1). In task 1, we use the [Libritts pretrained model](https://github.com/espnet/espnet/blob/master/egs/libritts/tts1/RESULTS.md#v050-first-multi-speaker-transformer-1024-pt-window--256-pt-shift--x-vector-with-add-integration--fast-gl-64-iters) provided by ESPnet. In task 2, corpora of two languages are used for pretraining: English and the language of the target speaker.
+3. TTS finetuning. We update all parameters using the training set of the target speaker.
+4. PWG training. We pool the training data from all available speakers in each task.
+
+## Recipe structure
+
+- `tts1_en_[de/fi/zh]`: TTS pretraining using English and the non-English language (de:German; fi:Finnish; zh:Mandarin).
+- `voc1`: PWG training.
+- `vc1_task[1/2]`: TTS finetuning, and the conversion phase for task 1 and task 2.
+
+## Requirements
+
+The following packages do not come with the installation of ESPnet. Please install them via `pip`.
+
+```
+cd <recipe>
+. ./path.sh
+pip install -U <package>
+```
+
+- [pypinyin](https://pypi.org/project/pypinyin/): used in `tts1_en_zh` and `vc1_task2`.
+- [parallel-wavegan](https://pypi.org/project/parallel-wavegan/): used in `voc1`, `vc1_task1` and `vc1_task2`.
+
+## Datasets and preparation.
+
+The following datasets are used to train this baseline method.
+
+- **LibriSpeech** contains English data for ASR training.
+- **LibriTTS** contains English data for task 1 TTS pretraining. It can be downloaded automatically in the recipe.
+- **M-AILABS** contains English and German data for task 2 TTS pretraining. It can be downloaded automatically in the recipe.
+- **[CSS10](https://www.kaggle.com/bryanpark/finnish-single-speaker-speech-dataset)** contains Finnish data for task2 TTS pretraining. To download this dataset, Kaggle membership is required. Please download and put it in the desired directory. (default is `tts_en_fi/downloads/`)
+- **CSMSC** contains Mandarin data for task 2 TTS pretraining. It can be downloaded automatically in the recipe.
+- **VCC2020** contains the main training data of the challenge. Please follow the instruction from the organizers and put in the desired directory. (default is `vc1_task[1/2]/downloads/`)
+
+## Usage
+
+Please see the readme in each recipe.
+
+## Author
+
+Main development: Wen-Chin Huang @ Nagoya University (2020/03)  
+Advisors: Tomoki Hayashi (Nagoya U), Shinji Watanabe (JHU), Tomoki Toda (Nagoya U)  
```

Comment:
Could we also add an acknowledgment to Matthew Wiesner for his help with phoneme transcription preparations?
E.g.,
```
Special thanks to Matthew Wiesner (JHU)
```
You could also add other people as well.